spinup_bis/algos/tf2/sop/sop.py

```diff
@@ -351,13 +305,7 @@ def learn_on_batch(obs1, obs2, acts, rews, done):
             q1_loss = tf.reduce_mean((q_backup - q1) ** 2, axis=0)
             q2_loss = tf.reduce_mean((q_backup - q2) ** 2, axis=0)
 
-            if use_weighted_bellman_backup:
-                q_std = tf.math.reduce_std(min_target_q, axis=0)
-                q_weight = tf.math.sigmoid(-q_std * bellman_temp) + 0.5
-
-                q_weight *= 0.5
-            else:
-                q_weight = 0.5
+            q_weight = 0.5
```

Comment:
Remove it and directly average the two losses below. See my suggestion below.

Reply:
done